<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="depositButton">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>81</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>存钱</string>
    </property>
   </widget>
   <widget class="QPushButton" name="withdrawButton">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>117</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>取钱</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transferButton">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>153</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>转账</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transactionButton">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>81</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="personalButton">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>117</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>个人信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="on_logoutButton">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>153</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
